spec/docile_spec.rb

```diff
@@ -435,6 +435,36 @@ def dsl_eval_string(string)
         end
       end
     end
+
+    if RUBY_VERSION >= "2.0.0"
+      context "when a DSL method has a keyword argument" do
+        class DSLMethodWithKeywordArgument
+          attr_reader :v0, :v1, :v2
+          class_eval(<<-METHOD)
+            def set(v0, v1:, v2:)
+              @v0 = v0
+              @v1 = v1
+              @v2 = v2
+            end
+          METHOD
```

Comment:
```suggestion

          def set(v0, v1:, v2:)
            @v0 = v0
            @v1 = v1
            @v2 = v2
          end
```

Reply:
@taichi-ishitani can you clarify if there's a reason to use `class_eval` meta-programming here?

Reply:
This method definition causes a syntax error on Ruby 1.8 and 1.9 because these rubies does not support keyword argument.
(see this CI log: https://travis-ci.com/github/taichi-ishitani/docile/jobs/460918552)
I need to use `class_eval` to avoid this syntax error.

Reply:
If Docile does not need to support Ruby 1.8 and 1.9 then I can remove `class_eval`.

Reply:
Ah! Thanks that makes sense, thank you for explaining. In that case, I'd like to merge as-is, and in the future when we drop 1.8 and 1.9, we can change it back.